### PR TITLE
add OOM datadog monitor

### PR DESCRIPTION
**Pull request by loafyloaf**

### CODE REVIEW

File: monitors.yml [hunk 1]
```diff
@@ -177,3 +177,24 @@ monitors:
...
+      {{#is_alert}}
+      less than 100M of memory on {{host.name}}
+      {{/is_alert}}
+
+      @zxkuqyb@gmail.com
+      @slack-pirateshiprevenge
+    multi: true
+    name: OOM on {{host.name}}
+    options:
+      notify_no_data: false
```
Comment (rattboi) [inline monitors.yml]: This seems like one where we should always get a response, no? So notifying on no data might be a good idea, as it also hints at other issues?

Comment (loafyloaf) [reply monitors.yml]: hrm.  a valid point, but until we are out of the free version - perhaps not a good idea?

(thanks for the review!)

Comment (loafyloaf) [reply monitors.yml]: updated that anyway - we'll deal with it otherwise if it becomes a problem

**Review by rattboi — APPROVED** [review]
:+1:

**Pull request merged**